### PR TITLE
Batiai 2296 - files for ESO

### PR DESCRIPTION
**Pull request by sshaikRC**
### Submitter Checklist
-  [ ] Is there an impact on Auditing and Logging procedures or capabilities?
-  [ ] Is there an impact on Authentication procedures or capabilities?
-  [ ] Is there an impact on Authorization procedures or capabilities?
-  [ ] Is there an impact on Communication Security procedures or capabilities?
-  [ ] Is there an impact on Cryptography procedures or capabilities?
-  [ ] Is there an impact on Sensitive Data procedures or capabilities?
-  [ ] Is there an impact on any other security-related procedures or capabilities?
-  [ ] No security impacts identified.

Security Risks Identified - For any applicable items on the "Submitter Checklist," describe the impact of the change and any implemented mitigations.

### CODE REVIEW

**Comment by robo-gotham** [comment]
Snyk Scanning for Commit: 7f529f8d341fe4534341981724520e390c932a0c

Snyk Infrastructure as Code

- Snyk testing Infrastructure as Code configuration issues.
✔ Test completed.

Issues
  No vulnerable paths were found!

-------------------------------------------------------

</details>

Test Summary

  Organization: batcave-ispg
  Project name: CMS-Enterprise/batcave-tf-eso

✔ Files without issues: 4
✗ Files with issues: 0
  Ignored issues: 0
  Total issues: 0 [ 0 critical, 0 high, 0 medium, 0 low ]

-------------------------------------------------------

Report Complete

  Your test results are available at: https://snyk.io/org/batcave-ispg/projects
  under the name: CMS-Enterprise/batcave-tf-eso

File: variables.tf [hunk 1]
```diff
@@ -8,14 +8,22 @@ variable "cluster_name" {
...
+
+variable "cluster_certificate_authority_data" {
+  description = "CA certificate data for EKS cluster"
+}
+
 variable "helm_namespace" {
   description = "Namespace for Helm chart"
-  default     = "kube-system"
+  default     = "external-secrets"
 }
```
Comment (bushong1) [inline variables.tf:22]: What creates this namespace?

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    cluster_ca_certificate = base64decode(var.cluster_certificate_authority_data)
+    token                  = data.aws_eks_cluster_auth.cluster.token
+  }
+}
+
+provider "kubernetes" {
+  host                   = var.cluster_endpoint
+  cluster_ca_certificate = base64decode(var.cluster_certificate_authority_data)
+  token                  = data.aws_eks_cluster_auth.cluster.token
+}
```
Comment (bushong1) [inline main.tf]: Not sure this is used, as there are no `kubernetes_` resources.  Can you delete an see if apply still works?

**Review by bushong1 — APPROVED** [review]
Small confirmation needed on the namespace creation, but approved otherwise.

File: variables.tf (hunk 1 above)
Comment (sshaikRC) [reply variables.tf:22]: Added a flag that creates the namespace if the namespace doesn't exist already.

File: main.tf (hunk 2 above)
Comment (sshaikRC) [reply main.tf]: Applied with this removed and worked ok so can remove this

**Pull request merged**